src/schemas/json/hammerkit.json

```diff
@@ -0,0 +1,349 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "definitions": {
+    "labels": {
+      "description": "A map of label values",
+      "type": "object",
+      "additionalProperties": { "type": "string" }
+    },
+    "envMap": {
+      "description": "A map of environment values",
+      "type": "object",
+      "additionalProperties": {
+        "type": "string"
+      }
+    },
+    "taskCommand": {
+      "anyOf": [
+        {
+          "description": "Command to be executed for the task",
+          "type": "string"
+        },
+        {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "cmd": {
+              "description": "Command to be executed for the task",
+              "type": "string"
+            },
+            "path": {
+              "description": "Path below build file path",
+              "type": "string"
+            }
+          },
+          "required": ["cmd"]
+        }
+      ]
+    },
+    "include": {
+      "type": "object",
+      "description": "A map of includes",
+      "additionalProperties": {
+        "type": "string"
+      }
+    },
+    "reference": {
+      "type": "object",
+      "description": "A map of references",
+      "additionalProperties": {
+        "type": "string"
+      }
+    },
+    "taskSource": {
+      "type": "array",
+      "description": "A list of source files and folders",
+      "items": {
+        "type": "string"
+      }
+    },
+    "taskGenerate": {
+      "type": "array",
+      "description": "A list of generated files and folders",
+      "items": {
+        "type": "string"
+      }
+    },
+    "taskDependency": {
+      "type": "array",
+      "description": "A list of task dependencies that get executed before the current task",
+      "items": {
+        "type": "string"
+      }
+    },
+    "taskNeed": {
+      "type": "array",
+      "description": "A list of task service needs that get ready before the current task",
+      "items": {
+        "type": "string"
+      }
+    },
+    "mount": {
+      "type": "array",
+      "description": "A list of file paths that get mounted into the container",
+      "items": {
+        "type": "string"
+      }
+    },
+    "taskPort": {
```

Comment:
Mandatory fix: [`pattern` property is missing](https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#best-practices).